daemon/src/main/java/org/mvndaemon/mvnd/daemon/ClientDispatcher.java

```diff
@@ -112,6 +113,19 @@ public void log(String msg) {
         queue.add(Message.log(trimTrailingEols(msg)));
     }
 
+    public void transfer(String projectId, TransferEvent e) {
+        int event = e.getType().ordinal();
```

Comment:
This looks fragile for the long term. Once they add/remove/reorder the members in `EventType`, our code stops working properly, but the errors will bubble up during the deserialization on the client side and they be hard to track back here. I think it would be safer to have some sort of explicit `int[]` (by EventType.ordinal()) or `EnumMap<EventType, Integer>` mapping to Message.type here.